core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/CodeSystemRequests.java

```diff
@@ -68,6 +69,14 @@ public static CodeSystemVersionCreateRequestBuilder prepareNewCodeSystemVersion(
 		return new CodeSystemVersionCreateRequestBuilder();
 	}
 	
+	// Upgrade API
+	
+	public static CodeSystemUpgradeRequestBuilder prepareUpgrade(CodeSystemURI codeSystem, CodeSystemURI extensionOf) {
```

Comment:
Please add an example with any restriction placed on the two parameters. 

If I got it right, the request will upgrade `codeSystem` so that it becomes an extension of `extensionOf`? And so the first argument needs to be a "bare" URI containing only a short name, whereas the second needs to have a version ID appended after the short name.

Reply:
Yes, the first URI is the `codeSystemId`, the second is the new dependency version URI.